EvidenceApi.Tests/V1/E2ETests/HealthCheckTests.cs

```diff
@@ -0,0 +1,32 @@
+using System;
```

Comment:
It's unclear why the `healthcheck/error` endpoint even exists. It seems like the easiest way to E2E test cross cutting functionality like this error handling but I'm more than happy to test an alternative endpoint if there is a good recommendation as this currently seems a bit hacky.

Reply:
As mentioned on the call, it looks like this endpoint was included as part of the repo template, so might be a Hackney convention.